manifest_staging/charts/gatekeeper/templates/gatekeeper-controller-manager-poddisruptionbudget.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
   name: gatekeeper-controller-manager
   namespace: '{{ .Release.Namespace }}'
 spec:
-  minAvailable: {{ .Values.pdb.controllerManager.minAvailable }}
+  minAvailable: {{ .Values.pdb.controllerManager | toYaml | nindent 2 }}
```

Comment:
changes in this file should be generated by running “make manifests” for updates in [manifest_staging](https://github.com/open-policy-agent/gatekeeper/tree/master/manifest_staging) directory

Reply:
okay. done.